vercel.json

```diff
@@ -0,0 +1,755 @@
+    },
+    {
+      "source": "/docs/plugins/fields/",
+      "destination": "/docs/fields",
+      "permanent": true
+    },
+    {
+      "source": "/docs/concepts/backends/",
+      "destination": "/docs/cms",
+      "permanent": true
+    },
+    {
+      "source": "/docs/concepts/plugins/",
+      "destination": "/docs/cms",
+      "permanent": true
+    },
+    {
+      "source": "/docs/getting-started/how-tina-works/",
+      "destination": "/docs/setup-overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/getting-started/overview/",
+      "destination": "/docs/setup-overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/getting-started/introduction/",
+      "destination": "/docs/setup-overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/getting-started/cms-set-up/",
+      "destination": "/docs/setup-overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/getting-started/edit-content/",
+      "destination": "/docs/setup-overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/getting-started/backends/",
+      "destination": "/docs/setup-overview/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/inline-editing/",
+      "destination": "/docs/ui/inline-editing",
+      "permanent": true
+    },
+    {
+      "source": "/docs/nextjs/inline-editing/",
+      "destination": "/docs/ui/inline-editing",
+      "permanent": true
+    },
+    {
+      "source": "/guides/nextjs/github-open-authoring/(.*)/",
+      "destination": "/docs/guides/",
+      "permanent": true
+    },
+    {
+      "source": "/docs/gatsby/inline-editing/",
+      "destination": "/docs/ui/inline-editing",
+      "permanent": true
+    },
+    {
+      "source": "/docs/inline-blocks/",
+      "destination": "/docs/ui/inline-editing/inline-blocks",
+      "permanent": true
+    },
+    {
+      "source": "/docs/inline-editing/inline-blocks/",
+      "destination": "/docs/ui/inline-editing/inline-blocks",
+      "permanent": true
+    },
+    {
+      "source": "/docs/inline-blocks/block-text/",
+      "destination": "/docs/ui/inline-editing/inline-text",
+      "permanent": true
+    },
+    {
+      "source": "/docs/inline-blocks/block-textarea/",
+      "destination": "/docs/ui/inline-editing/inline-textarea",
+      "permanent": true
+    },
+    {
+      "source": "/docs/inline-blocks/block-image/",
+      "destination": "/docs/ui/inline-editing/inline-image",
+      "permanent": true
+    },
+    {
+      "source": "/docs/nextjs/bootstrapping/",
+      "destination": "/guides/nextjs/git/getting-started",
+      "permanent": true
+    },
+    {
+      "source": "/docs/nextjs/adding-backends/",
+      "destination": "/guides/nextjs/git/adding-backend",
+      "permanent": true
+    },
+    {
+      "source": "/docs/nextjs/creating-forms/",
+      "destination": "/guides/nextjs/git/creating-git-forms",
+      "permanent": true
+    },
```

Comment:
I dont think these are working due to that other redirect we added.

```json
    {
      "source": "/guides/",
      "destination": "/docs/guides/",
      "permanent": true
    },  
```

I think this should be the last rule thats executed so it catches all the `/guides/{{SOME PATH}}` and makes them `/docs/guides/{{SOME PATH}}`

Reply:
As per our conversation, you move the catchall down to the bottom

Reply:
I think this is possible – tbc